packages/frontend/src/components/atoms/alert.atom.tsx

```
@@ -0,0 +1,67 @@
+ */
+export function Alert({ status = 'info', title, description, icon, className, children }: AlertProps) {
+  const statusStyles = {
+    info: 'bg-blue-50 border-blue-200 text-blue-800',
+    warning: 'bg-yellow-50 border-yellow-200 text-yellow-800',
+    error: 'bg-red-50 border-red-200 text-red-800',
+    success: 'bg-green-50 border-green-200 text-green-800',
+  };
+
+  const defaultIcons = {
+    info: (
+      <svg className="h-5 w-5" fill="currentColor" viewBox="0 0 20 20">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)